src/components/About.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import solution1 from '../images/solution1.svg';
 import solution2 from '../images/solution2.svg';
 import solution3 from '../images/solution3.svg';
-import ContactButton from './ContactButton';
+import Button from './Button';
+import NavBar from './NavBar';
 
 function About() {
   return (
     <section className="about">
+      <NavBar />
```

Comment:
also you can add `<NavBar/>` in upper layout in your application, what would don't added in each component when it need